addons/new-osio-addon/src/main/java/io/fabric8/launcher/osio/steps/GitSteps.java

```diff
@@ -80,7 +80,7 @@ public void createWebHooks(OsioProjectile projectile, GitRepository gitRepositor
         String jenkinsWebhookURL = EnvironmentVariables.ExternalServices.getJenkinsWebhookURL();
         try {
             // TODO: Check if the webhook requires a secret
-            gitService.createHook(gitRepository, null, new URL(jenkinsWebhookURL));
+            gitService.createHook(gitRepository, null, new URL(jenkinsWebhookURL), gitService.getSuggestedNewHookEvents());
```

Comment:
Is this change really needed? The `gitService.getSuggestedNewHookEvents());` is the default value if nothing is provided, right?

Reply:
Indeed I can remove it now it's part of the doc:
```
     * @param events     - the events that trigger the webhook; if none specified, {@see getSuggestedNewHookEvents} is used by default.
```